scrapy/downloadermiddlewares/retry.py

```diff
@@ -121,23 +128,7 @@ def parse(self, response):
     return None
 
 
-class RetryMiddleware:
-    # IOError is raised by the HttpCompression middleware when trying to
-    # decompress an empty response
-    EXCEPTIONS_TO_RETRY = (
-        defer.TimeoutError,
-        TimeoutError,
-        DNSLookupError,
-        ConnectionRefusedError,
-        ConnectionDone,
-        ConnectError,
-        ConnectionLost,
-        TCPTimedOutError,
-        ResponseFailed,
-        IOError,
-        TunnelError,
-    )
-
+class RetryMiddleware(six.with_metaclass(BackwardsCompatibilityMetaclass, object)):
```

Comment:
I *think* this should be `class RetryMiddleware(metaclass=BackwardsCompatibilityMetaclass)`